packages/@aws-cdk/aws-ivs/lib/channel.ts

```diff
@@ -152,17 +153,19 @@ export class Channel extends ChannelBase {
 
   constructor(scope: Construct, id: string, props: ChannelProps = {}) {
     super(scope, id, {
-      physicalName: props.name,
+      physicalName: props.channelName ?? Lazy.string({
```

Comment:
We should only generate a physical name for you if the resource requires that a name be set. In the case of `CfnChannel` the name is an optional property.

> Use generated resource names, not physical names

This is instructing you to leave the `name` undefined so that CloudFormation will generate a physical name for you.

Reply:
Thank you very much for reviewing this PR @corymhall !!

Please tell me about the CDK physical name design. I understood physical name is generated by the CDK or CloudFormation. However, `aws-ivs.Channel` not generate a physical name both CDK and CloudFormation. So when we not provide physicalName then channel is created by CloudFormation with no name.
So, rather than having CloudFormation generate the physical names, you mean that we should leave the behavior of CloudFormation up to whether it generates the physical names or not when we do not provide them, right?

Reply:
@WinterYukky that is the case for playback key pair as well?

Reply:
@corymhall 
Good that there is no discrepancy in the perception of the physical name design policy.
Yes, the same applies to playback key pairs.

<img width="1084" alt="image" src="https://user-images.githubusercontent.com/49480575/226978158-dfcf80df-9f96-499c-a340-faa85e779793.png">

<img width="1085" alt="image" src="https://user-images.githubusercontent.com/49480575/226977883-8247e58f-c45c-4073-ba0f-756b65e20080.png">